src/components/application_manager/rpc_plugins/app_service_rpc_plugin/src/app_service_app_extension.cc

```diff
@@ -112,6 +116,18 @@ void AppServiceAppExtension::ProcessResumption(
     for (size_t i = 0; i < subscriptions_app_services.length(); ++i) {
       std::string service_type = subscriptions_app_services[i].asString();
       SubscribeToAppService(service_type);
+
+      auto gasd_subscribe = app_mngr::MessageHelper::CreateMessageForHMI(
+          hmi_apis::messageType::request,
+          app_manager_->GetNextHMICorrelationID());
+      (*gasd_subscribe)[app_mngr::strings::params]
+                       [app_mngr::strings::function_id] =
+                           hmi_apis::FunctionID::AppService_GetAppServiceData;
+      (*gasd_subscribe)[app_mngr::strings::msg_params]
```

Comment:
@iCollin I believe you should implement `RevertResumption` as well because in your implementation I see that service may resume subscription, but in failure case it won't be reverted (unsubscribed vie GASD(subscribe=false) )

Reply:
Currently I am only budgeted time to review 0190 contribution as this our last sprint of development.

This will have to be done by 0190 contributor.